src/dotenv_vault/main.py

```diff
@@ -3,13 +3,28 @@
 from base64 import b64decode
 import io
 import os
+import logging
+
 from typing import (IO, Optional, Union)
 from urllib.parse import urlparse, parse_qsl
 
 from cryptography.hazmat.primitives.ciphers.aead import AESGCM
 from cryptography.exceptions import InvalidTag
 import dotenv.main as dotenv
 
+logger = logging.getLogger(__name__)
+
+def load_dotenv_vault() -> str:
+    path = dotenv.find_dotenv()
+    if not path:
+        return path
+    path = os.path.dirname(path)
+    if '.env.vault' not in os.listdir(path):
+        # we fall back to .env
+        logger.warning("No .env.vault file found. Falling back to .env")
```

Comment:
Messaging should match the dotenv messaging.

https://github.com/motdotla/dotenv/blob/master/lib/main.js#L228C113-L228C113

Reply:
done